www/docs/src/content/docs/0.1.0-beta.5/config-reference/dashboard.md

```diff
@@ -0,0 +1,166 @@
+---
+title: dashboardConfig
+description: A reference page for dashboard
+sidebar:
+  order: 4
+slug: 0.1.0-beta.5/config-reference/dashboard
+---
+
+`dashboardConfig` option let's customize the dashboard for the `astroStudioCMS`. This is used to setup your dashboard data.
+
+## Usage
+
+```js title="astro.config.mjs"  {2-24}
+astroStudioCMS({
+  dashboardConfig: {
+    dashboardEnabled: true,
+    faviconURL: "/favicon.ico",
+    dashboardRouteOverride: "/components/dashboard",
+    AuthConfig: {
+        providers: {
+            github: true,
+            usernameAndPassword: true
+        },
+        enabled: true
+    },
+    UnoCSSConfigOverride: {
+        injectReset: true,
+        injectEntry: true,
+        presetsConfig: {
+            presetDaisyUI: {
+                themes: ['dark', 'light'],
+                lightTheme: ['light'],
+                darkTheme: ['dark'],
+            }
+        }
+    }
+  }, // DEFAULT - This injects a dashboard to setup your dashboard data.
+})
+```
+
+### `dashboardEnabled`
+
+* **Type:** `boolean | undefined`
+* **Default:** `true`
+
+Allows the user to enable or disable the Astro StudioCMS dashboard but still provide all the helper's and utilities to those who are customizing their setup, doing so will disable the dashboard and you will need to manage your content via the Astro Studio Dashboard at [Astro Studio](http://studio.astro.build).
```

Comment:
```suggestion
Allows enabling or disabling of the Astro StudioCMS dashboard but still provide all the helpers and utilities to those who are customizing their setup. By disabling it will disable the dashboard and you will need to manage your content via the Astro Studio Dashboard at [Astro Studio](http://studio.astro.build).
```

Reply:
This is the old version of the docs.... (beta.5)

Reply:
Updated the Latest version of the docs on [d869f86](https://github.com/astrolicious/studiocms/pull/281/commits/d869f869670c49fbd9c8f357a32ed15fe00a20c5)